docs/architecture/dapr-for-net-developers/getting-started.md

```diff
@@ -22,7 +22,7 @@ You'll start by installing Dapr on your development computer. Once complete, you
 
 1. [Initialize Dapr](https://docs.dapr.io/getting-started/install-dapr/). This step sets up your development environment by installing the latest Dapr binaries and container images.
```

Comment:
Lines 20 and 21 need to be indented 3 spaces each, otherwise the numbering for the bullets restarts - see https://review.docs.microsoft.com/en-us/dotnet/architecture/dapr-for-net-developers/getting-started?branch=pr-en-us-27146#install-dapr-into-your-local-environment.

Reply:
I've added the indentation.